source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -343,48 +343,62 @@ def add_pt_sections(journey):
+    assert get_kraken_calls(req) == {("walking", "walking", "only"), ("bike", "walking", "only")}
 
-    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["bss"]}
-    assert get_kraken_calls(req) == {("bike", "bss")}
+    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["bss"], "direct_path": "indifferent"}
+    assert get_kraken_calls(req) == {("bike", "bss", "indifferent")}
 
     req = {"origin_mode": ["bike", "car"], "destination_mode": ["bss"]}
-    assert get_kraken_calls(req) == {("bike", "bss"), ("car", "bss")}
+    assert get_kraken_calls(req) == {("bike", "bss", "indifferent"), ("car", "bss", "indifferent")}
 
     req = {"origin_mode": ["bss", "bike"], "destination_mode": ["bike"]}
-    assert get_kraken_calls(req) == {("bike", "bike")}
+    assert get_kraken_calls(req) == {("bike", "bike", "indifferent")}
 
     req = {"origin_mode": ["taxi", "bike"], "destination_mode": ["walking", "bss", "car", "ridesharing", "taxi"]}
     assert get_kraken_calls(req) == {
-        ('taxi', 'taxi'),
-        ('taxi', 'bss'),
-        ('bike', 'bss'),
-        ('bike', 'walking'),
-        ('bike', 'taxi'),
-        ('taxi', 'walking'),
-        ('taxi', 'ridesharing'),
+        ('taxi', 'taxi', "indifferent"),
+        ('taxi', 'bss', "indifferent"),
+        ('bike', 'bss', "indifferent"),
+        ('bike', 'walking', "indifferent"),
+        ('bike', 'taxi', "indifferent"),
+        ('taxi', 'walking', "indifferent"),
+        ('taxi', 'ridesharing', "indifferent"),
     }
 
     req = {
         "origin_mode": ["ridesharing", "taxi"],
         "destination_mode": ["walking", "bss", "bike", "car", "ridesharing", "taxi"],
     }
     assert get_kraken_calls(req) == {
-        ('taxi', 'taxi'),
-        ('ridesharing', 'walking'),
-        ('taxi', 'bss'),
-        ('ridesharing', 'bss'),
-        ('taxi', 'ridesharing'),
-        ('taxi', 'walking'),
-        ('ridesharing', 'taxi'),
+        ('taxi', 'taxi', "indifferent"),
+        ('ridesharing', 'walking', "indifferent"),
+        ('taxi', 'bss', "indifferent"),
+        ('ridesharing', 'bss', "indifferent"),
+        ('taxi', 'ridesharing', "indifferent"),
+        ('taxi', 'walking', "indifferent"),
+        ('ridesharing', 'taxi', "indifferent"),
     }
 
+    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"], "direct_path_mode": ["walking"]}
+    assert get_kraken_calls(req) == {("walking", "walking", "indifferent")}
+
+    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"], "direct_path_mode": ["bike"]}
+    assert get_kraken_calls(req) == {("walking", "walking", "indifferent"), ("bike", "bike", "only")}
+
+    req = {
+        "origin_mode": ["bss", "walking"],
+        "destination_mode": ["walking"],
+        "direct_path_mode": ["bike"],
+        "direct_path": "none",
+    }
+    assert get_kraken_calls(req) == {("walking", "walking", "none"), ("bike", "bike", "only")}
```

Comment:
Here we could save a call and remove the (bike, bike, only) as it would be filtered by the `pt=none` filter.